src/main/java/com/company/Main.java

```diff
@@ -75,15 +82,37 @@ public static void printOptions() {
     public static void createPersonWithAllAttributes() {
 
         CreatePersonService createPersonService = new CreatePersonService();
-
         Person newPerson = new Person(createPersonService.createPersonName(), createPersonService.createPersonGender(), createPersonService.createPersonDateOfBirth(), createPersonService.createPersonInterests(), createPersonService.createPersonStatus());
-        if (personService.addNewPerson(newPerson)) {
-            System.out.println(newPerson.toString());
-        } else {
-            System.out.println(CANNOT_PERFORM_ACTION);
+
+
+        try (FileWriter fileWriter = new FileWriter("personsList.txt", true);
+             BufferedWriter bufferedWriter = new BufferedWriter(fileWriter)){
+
+            if (personService.addNewPerson(newPerson)) {
+                bufferedWriter.write(newPerson.toString());
+                bufferedWriter.newLine();
+                bufferedWriter.append(System.lineSeparator());
+                bufferedWriter.close();
```

Comment:
Isn't the writer closed automatically ? You are using try-with-resources. If you decide to use the classic approach this should be in the **finally** block